pwndbg/commands/hexdump.py

```diff
@@ -29,18 +27,17 @@
 )
 
 
-def address_or_module_name(s):
-    gdbval_or_str = pwndbg.commands.sloppy_gdb_parse(s)
-    if isinstance(gdbval_or_str, str):
-        module_name = gdbval_or_str
-        pages = list(filter(lambda page: module_name in page.objfile, pwndbg.gdblib.vmmap.get()))
+def address_or_module_name(s) -> int:
+    addr_or_str: int | str = pwndbg.commands.sloppy_gdb_parse(s)
+    if isinstance(addr_or_str, str):
+        module_name = addr_or_str
+        pages = list(filter(lambda page: module_name in page.objfile, pwndbg.aglib.vmmap.get()))
         if pages:
             return pages[0].vaddr
         else:
             raise argparse.ArgumentTypeError(f"Could not find pages for module {module_name}")
-    elif isinstance(gdbval_or_str, (int, gdb.Value)):
-        addr = gdbval_or_str
-        return addr
+    elif isinstance(addr_or_str, int):
+        return addr_or_str
```

Comment:
Do I understand correctly that now the `sloppy_gdb_parse` returns `int` or `str` and never a gdb.value? as in the value is resolved to int already there?

Reply:
Random thought: the `sloppy_gdb_parse` name is not so great..